.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+Documentation:
+  Enabled: false
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
We went with Thoughtbot's reasoning on this, which was that there are no performance benefits to double vs single and therefore you don't have to think about escaping or interpolating.